src/test/java/org/gridsuite/modification/server/modifications/BatteryModificationTest.java

```diff
@@ -81,6 +83,17 @@ protected void assertNetworkAfterCreation() {
         assertEquals(0.1f, modifiedBattery.getExtension(ActivePowerControl.class).getDroop());
         assertEquals(true, modifiedBattery.getExtension(ActivePowerControl.class).isParticipate());
         assertEquals(ReactiveLimitsKind.CURVE, modifiedBattery.getReactiveLimits().getKind());
+        Collection<ReactiveCapabilityCurve.Point> points = modifiedBattery.getReactiveLimits(ReactiveCapabilityCurve.class).getPoints();
+        List<ReactiveCapabilityCurveModificationInfos> modificationPoints = batteryModificationInfos.getReactiveCapabilityCurvePoints();
+        if (!CollectionUtils.isEmpty(points)) {
```

Comment:
Why using batteryModificationInfos.getReactiveCapabilityCurvePoints() here ?
Assertions on the battery should just check the points collection, no ?